### PR TITLE
Update about section

### PR DESCRIPTION
**Pull request by chr4**
- [ ] AppStore links
- [ ] White label updates
- [ ] Pictures and Videos for the real-world example
- [ ] About section might be not prominent enough?
- [ ] Highlight new features and changes since last release (CHANGELOG)

### CODE REVIEW

File: src/views/ProductDescription/ProductDescription.js [hunk 1]
```diff
@@ -135,20 +137,27 @@ export default class ProductDescription extends Component {
               This way, the issuer can provide a simple setup routine for the
               customer by generating a QR code, that the customer can scan with
               the mobile wallet.
```
Comment (chr4) [inline src/views/ProductDescription/ProductDescription.js:162]: This is going to be obsolete once the white label apps are in place, right?

Comment (fabrik42) [reply src/views/ProductDescription/ProductDescription.js:162]: Yes, I think so :)

**Review by fabrik42 — APPROVED** [review]

**Comment by fabrik42** [comment]
> About section might be not prominent enough?

Yes, you are right, not happy with this either.

**Comment by fabrik42** [comment]
I would like merge this and try to work on this problem afterwards:

> About section might be not prominent enough?

**Comment by chr4** [comment]
I'd leave it open because we could work in some changes.

**Comment by fabrik42** [comment]
Ok :) So we wait until the other one is resolved.

**Comment by chr4** [comment]
Ok, I've put in most of the things, although there are some TODOs left. 
Most of them rely on the new about structure though.

**Pull request merged**